### PR TITLE
feat: adding simple abci service

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Adds the simple abci service.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: packages/valory/services/simple_abci/service.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: simple_abci
```
Comment (dvilelaf) [inline packages/valory/services/simple_abci/service.yaml:1]: I'm trying to locally run the service to check it:

```
autonomy init --reset --local
autonomy deploy build deployment valory/simple_abci:bafybeif5infufz5m2ryh26e6vhmh5sqyejeeizarcsbazkktujfnymvcee keys.json
cd abci_build/
docker-compose up --force-recreate
```

And when the agents start I get this:
```
Starting AEA 'agent' in 'async' mode...
[2022-08-04 11:18:44,112] [ERROR] [agent] Error while connecting <class 'connection_module.ABCIServerConnection'>: FileNotFoundError(2, 'No such file or directory')
```

@angrybayblade any idea?

File: packages/valory/agents/simple_abci/aea-config.yaml [hunk 2]
```diff
@@ -59,6 +59,7 @@ public_id: valory/abci:0.1.0
 type: connection
 config:
   target_skill_id: valory/simple_abci:0.1.0
+  use_tendermint: ${ABCI_USE_TENDERMINT:bool:false}
```
Comment (dvilelaf) [inline packages/valory/agents/simple_abci/aea-config.yaml:62]: I'm verifying that we really need this.

File: tests/test_docs/test_code_blocks.py [hunk 3]
```diff
@@ -174,7 +174,7 @@ class TestPythonSnippets(BaseTestDocCode):
                 "packages/valory/skills/hello_world_abci/payloads.py",
             ],
         },
-        "docs/price_oracle_intro.md": {
+        "docs/price_oracle_technical_details.md": {
```
Comment (dvilelaf) [inline tests/test_docs/test_code_blocks.py:177]: This had been renamed

File: Makefile [hunk 4]
```diff
@@ -14,6 +14,7 @@ clean-build:
 	rm -fr pip-wheel-metadata
 	find . -name '*.egg-info' -exec rm -fr {} +
 	find . -name '*.egg' -exec rm -fr {} +
+	find . -type d -name __pycache__ -exec rm -rv {} +
```
Comment (dvilelaf) [inline Makefile:17]: I've added this to also delete the __pycache__ folders, as it is handy before pushing packages to ipfs so the CLI does not complain.

**Review by angrybayblade — APPROVED** [review]
LGTM

File: docs/price_oracle_intro.md [hunk 5]
```diff
@@ -64,7 +64,7 @@ you have followed the [setup instructions](quick_start.md#setup). As a result yo
 
 3. Use the {{open_autonomy}} CLI to download and build the agent images:
     ```bash
-    autonomy deploy build deployment valory/oracle_hardhat:bafybeihurssdozdmslwc5v5tfwnefflobe7zrxgyuruc5g76khc6l6mkke keys.json
+    autonomy deploy build deployment valory/oracle_hardhat:0.1.0:bafybeicrjeiaxcqsas66cpheh4ucptbftz7lvnlau43p62gliap45cet6a keys.json
```
Comment (angrybayblade) [inline docs/price_oracle_intro.md:67]: Why did this hash change?

Comment (dvilelaf) [reply docs/price_oracle_intro.md:67]: The base PR had outdated has in the docs: https://github.com/valory-xyz/open-autonomy/pull/1152

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**